libcxx/test/std/iterators/iterator.requirements/iterator.cust/iterator.cust.move/iter_move.pass.cpp

```diff
@@ -186,6 +193,9 @@ template<class T> struct Holder { T t; };
 static_assert(std::is_invocable_v<IterMoveT, Holder<Incomplete>**>);
 static_assert(std::is_invocable_v<IterMoveT, Holder<Incomplete>**&>);
 
+// Ordinary unqualified lookup should not be performed.
+static_assert(!std::is_invocable_v<IterMoveT, ordinary_unqualified_lookup_helpers::StructWithGlobalIterMove&>);
```

Comment:
not a big issue but usually for this kind of tests, we put them into a single file. e.g.
ranges_robust_against_*.pass.cpp

what do you think?

Reply:
I think this is good idea, I've added new three files `robust_against_ordinary_unqualified_lookup.compile.pass.cpp`.

Commit: https://github.com/llvm/llvm-project/pull/74534/commits/510a91a9fe1301da6a95f82328c47a9959a35ee9